canonical-web-and-design/ubuntu.com
# Create /appliances/vm page

## Done

- Create /appliances/virtual-machines page
- Added a few strips including one for the /appliance page when we get there
- renamed _docs/ as '_' files don't render anymore to 0_docs/
- Hide /appliance/docs and /appliance/tutorials for now

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliances/virtual-machines
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [design](https://app.zeplin.io/project/5eb3d16a91c4117789940141/screen/5ee24a3e2150d5b8fdea8086) and the [copy doc](https://docs.google.com/document/d/1AdLZ6jhGhKaYg6TSnR_Xl8ivRrburrsA8161t_Pk2q8/edit#)
